dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -233,6 +233,7 @@ def test_add_profilers(self, *mocks):
         self.assertEqual(
             "<class 'pandas.core.frame.DataFrame'>", merged_profile.file_type
         )
+        self.assertTrue(merged_profile.options.row_statistics.null_count.is_enabled)
```

Comment:
This is a fair check when doing merge as it `__add__` should ensure the consistency.